src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/TasksController.kt

```diff
@@ -89,9 +89,53 @@ class TasksController(
     val tasks = mutableListOf<Task>()
 
     if (user.hasRole(UserRole.CAS1_MATCHER)) {
-      async { tasks += getPlacementRequestTasks(placementRequestService.getVisiblePlacementRequestsForUser(user), user) }
+      async {
+        tasks += getPlacementRequestTasks(
+          placementRequestService.getVisiblePlacementRequestsForUser(user),
+          user,
+        )
+      }
 
-      async { tasks += getPlacementApplicationTasks(placementApplicationService.getVisiblePlacementApplicationsForUser(user), user) }
+      async {
+        tasks += getPlacementApplicationTasks(
+          placementApplicationService.getVisiblePlacementApplicationsForUser(user),
+          user,
+        )
+      }
+    }
+
+    return@runBlocking ResponseEntity.ok(tasks)
+  }
+
+  override fun tasksTaskTypeGet(taskType: String): ResponseEntity<List<Task>> = runBlocking {
+    val user = userService.getUserForRequest()
+    val tasks = mutableListOf<Task>()
+    val type = enumConverterFactory.getConverter(TaskType::class.java).convert(
+      taskType.kebabCaseToPascalCase(),
+    ) ?: throw NotFoundProblem(taskType, "TaskType")
+
+    if (user.hasRole(UserRole.CAS1_MATCHER)) {
+      when (type) {
+        TaskType.placementApplication -> {
+          async {
+            tasks += getPlacementApplicationTasks(
```

Comment:
We probs don't need to push into a list here, given that the list will only contain one type of task